src/pages/dashboard.tsx

```diff
@@ -342,9 +342,16 @@ const DashboardContent: React.FC<DashboardContentProps> = ({
   pluginAddress,
 }) => {
   const {transfers, totalAssetValue} = useDaoVault();
-  const {data: proposals} = useProposals(daoAddressOrEns, pluginType);
 
-  const proposalCount = proposals.length;
+  const {data: proposals} = useProposals({
+    daoAddressOrEns,
+    pluginType,
+  });
```

Comment:
Instead of calling `proposals?.pages.flat() ?? []` below multiple times, I would define this as a variable:
```suggestion
  const {data: proposalsData} = useProposals({
    daoAddressOrEns,
    pluginType,
  });
  const proposals = proposalsData?.pages.flat() ?? [];
```

Reply:
Fair point